src/routes.json

```diff
@@ -6,6 +6,14 @@
         "view": "about/about",
         "title": "About"
     },
+    {
+        "name": "annual-report",
+        "pattern": "^/annual-report/?$",
+        "routeAlias": "/annual-report/?$",
+        "view": "annual-report/annual-report",
+        "title": "annual-report",
+        "viewportWidth": "device-width"
```

Comment:
I'm not sure if we need this or exactly what it does. We added it when we were trying to fix issues we were having with responsiveness.